### PR TITLE
update footer

### PR DESCRIPTION
**Pull request by wuyongyu**
update footer

### CODE REVIEW

**Comment by wuyongyu** [comment]
before:
image

---
after
image

File: themes/docsy/layouts/partials/footer.html [hunk 1]
```diff
@@ -2,26 +2,28 @@
 <footer class="bg-dark py-5 row d-print-none">
   <div class="container-fluid mx-sm-5">
     <div class="row">
-      <div class="col-6 col-sm-4 text-xs-center order-sm-2">
+      <!-- <div class="col-6 col-sm-4 text-xs-center order-sm-2">
```
Comment (javeme) [inline themes/docsy/layouts/partials/footer.html:5]: we can remove the code if don't need anymore

Comment (wuyongyu) [reply themes/docsy/layouts/partials/footer.html]: ok

**Review by javeme — APPROVED** [review]
LGTM

**Comment by wuyongyu** [comment]
license link: https://www.apache.org/foundation/marks/pmcs

image

**Review by imbajin — APPROVED** [review]

**Pull request merged**